changelog/7539.feature.rst

```diff
@@ -0,0 +1,2 @@
+add feature to ignore private names(characterised with names starting with '_') from being collected when running pytest
```

Comment:
"add feature to " is redundant

I would probably write less:
```
``-k``: don't match names starting with ``_``.
```

Reply:
Included the suggestged change in 6b5fbe9

Reply:
you didn't take my suggestion, there's still a bunch of extraneous text